src/client/search_results.py

```diff
@@ -50,182 +50,82 @@ def _create_search_result_class_for(type_: Type, docstring: Optional[str] = None
-    """
+    _create_search_result_docstring('training pools')
 )
 UserBonusSearchResult = _create_search_result_class_for(
     UserBonus,
-    """The list of found `UserBonus` instances and whether there is something else on the original request
-
-    It's better to use TolokaClient.get_user_bonuses(), which already implements the correct handling of the search result.
-
-    Attributes:
-        items: List of found `UserBonus` instances
-        has_more: Whether the list is complete:
-            * True - Not all elements are included in the output due to restrictions in the limit parameter.
-            * False - The output lists all the items.
-    """
+    _create_search_result_docstring("rewards", "Tolokers' rewards")
 )
 UserRestrictionSearchResult = _create_search_result_class_for(
     UserRestriction,
-    """The list of found Toloker restrictions and whether there is something else on the original request
-
-    It's better to use TolokaClient.get_user_restrictions(), which already implements the correct handling of the search result.
-
-    Attributes:
-        items: List of found Toloker restrictions
-        has_more: Whether the list is complete:
-            * True - Not all elements are included in the output due to restrictions in the limit parameter.
-            * False - The output lists all the items.
-    """
+    _create_search_result_docstring('Toloker restrictions')
 )
 UserSkillSearchResult = _create_search_result_class_for(
     UserSkill,
-    """The list of found Toloker skills and whether there is something else on the original request
-
-    It's better to use TolokaClient.get_user_skills(), which already implements the correct handling of the search result.
-
-    Attributes:
-        items: List of found Toloker skills
-        has_more: Whether the list is complete:
-            * True - Not all elements are included in the output due to restrictions in the limit parameter.
-            * False - The output lists all the items.
-    """
+    _create_search_result_docstring("skills", "Tolokers' skills")
 )
 WebhookSubscriptionSearchResult = _create_search_result_class_for(
     WebhookSubscription,
-    """The list of found subscriptions and whether there is something else on the original request
+    _create_search_result_docstring('subscriptions', 'webhook subscriptions')
+)
+OperationSearchResult = _create_search_result_class_for(
+    Operation,
+    """The list of found operations and whether there is something else on the original request
 
-    It's better to use TolokaClient.get_webhook_subscriptions(),
+    It's better to use TolokaClient.get_operations(),
     which already implements the correct handling of the search result.
 
     Attributes:
-        items: List of found subscriptions
+        items: List of found operations
         has_more: Whether the list is complete:
             * True - Not all elements are included in the output due to restrictions in the limit parameter.
```

Comment:
I think you should highlight these ones as well as those in lines 60-61 (with backtics)

Reply:
Thanks. 
(Missed this change after pulling from master.)